### DISCUSSION TITLE
[Feature] Batch Image Download to Mobile Device

### DISCUSSION THREAD

**Discussion by skycryer** [immich-app/immich#13054] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Images that have been uploaded to the server and delete from the device, like mobile device can be downloaded if you open the image. But the button for "download to device" is only available per image. But maybe I want to download a bunch off images to be on device for offline usage. 

So please make the download button available also when you select multiple images. Maybe on the same list where you can bulk delete images or favourite them.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5068
